.github/workflows/spark.yaml

```diff
@@ -0,0 +1,28 @@
+name: spark
+on:
+  push:
+    paths:
+      - "clients/spark/**"
+    branches:
+      - master
+  pull_request:
+jobs:
+  lakefs-spark:
+    name: lakeFS Spark
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check-out code
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+
+      - name: package
+        working-directory: clients/spark
+        run: sbt package
```

Comment:
Can we compile twice, once with 2.11 and once with 2.12?  Because we *do* need to support both.  (I think all Scala compilers take a language version command-line flag, so you will *not* have to install a separate Scala compiler.)